src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -700,6 +714,23 @@ function CreatePipelineRun({ intl }) {
               defaultMessage: 'Enabled'
             })}
           />
+          <TextInput
+            id="create-pipelinerun--pipelinerunname"
+            labelText={intl.formatMessage({
+              id: 'dashboard.createRun.pipelineRunNameLabel',
+              defaultMessage: 'PipelineRun name'
+            })}
+            invalid={validationError && !validPipelineRunName}
+            invalidText={intl.formatMessage({
+              id: 'dashboard.createRun.invalidRunName',
```

Comment:
Can we repurpose the existing string for this?

Rename `dashboard.createPipelineResource.nameError` to `dashboard.createResource.nameError` and use it here